packages/react/src/components/NumberInput/NumberInput.tsx

```diff
@@ -209,8 +209,8 @@ const NumberInput = React.forwardRef(function NumberInput(
     invalid = false,
     invalidText,
     light,
-    max = 100,
-    min = 0,
+    max,
```

Comment:
```suggestion
    max = Infinity,
```

Would this be a safe middle ground between the original issue and the breaking change?

Reply:
Kind of. Even with this the `clamp` function will still return `NaN` if `min` is undefined.